lib/PuppeteerSharp/ConsoleMessage.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+
+namespace PuppeteerSharp
+{
+    public class ConsoleMessage
+    {
+        public ConsoleType Type { get; }
+        public IList<JSHandle> Args { get; }
+
+        public ConsoleMessage(ConsoleType type, IList<JSHandle> args)
```

Comment:
ConsoleMessage has a text property which is different from the args

https://github.com/GoogleChrome/puppeteer/blob/v1.0.0/lib/Page.js#L447

Reply:
Yes, at the beginning it was present, I didn't see the purpose ... I replaced it by this in ConsoleMessage which does the same 

```cs
    public string Text() => string.Join(" ", Args);
```

If really needed I'll change it

Reply:
Yup, we have to honor Puppeteer